mobile/ethclient.go

```diff
@@ -198,8 +198,8 @@ func (ec *EthereumClient) FilterLogs(ctx *Context, query *FilterQuery) (logs *Lo
 	}
 	// Temp hack due to vm.Logs being []*vm.Log
 	res := make([]*types.Log, len(rawLogs))
-	for i, log := range rawLogs {
-		res[i] = &log
+	for i, _ := range rawLogs {
```

Comment:
:+1: but why to keep blank identifier?

Reply:
I'm not a go developer and didn't know that return parameters are optional.
Removed it and the code is compiled.
I'll update the PR soon.